docs/edit/manifest.md

```diff
@@ -0,0 +1,130 @@
+### Context
+
+The usual way to declare what will be tested or not are decorators (`@released`, `@bug`, `@missing_feature`...). This solution offers several advantages :
+
+- declarations are as close as possible to the test, making the link obvious
+- and complex condition (several components involved, complex version range...) is easy to do, as it's declared as python code
+
+### Issue
+
+Unfortunatly, it comes with a major drawback: Activating a single test can become a hell due to confilct between PRs.
+
+### Proposition
+
+To solve this, we'll try to leverage two points :
+
+1. Often, those declarations involve only one component
+2. Often, a component is modified by only one developper
+
+The idea is to offer an alternative way to declare those metadata, using one file per component, also known as manifest file. Those files will be inside system tests repo, declaring for a given component some metadata.
+
+Pro :
+
+- Will solve the majority of conflict during test activation PRs
+- each team will have ownership of its manifest file, easing the PR preview process
+
+Cost :
+
+- declarations will be a little bit far away from the test, asking a small addition in the learning curve
+- re-naming will be a little bit harder, and will spam all teams
+
+## Implementation
+
+- Manifests files will be inside `utils/manifest/`, one per term, named with the argument name used in decorators (`utils/manifest/golang.yml`, `utils/manifest/agent.yml`...)
+- Each team will have ownership of its manifest file
+- pytest will load those files, and decorate tests node during collection
+
+## Supported features
+
+Will suport:
```

Comment:
IMHO this should land together with a script that:
* Checks if manifest files are in sync with tests (e.g. not referencing missing classes).
* Ideally, also a way to add missing entries based on tests (or a way to list tests disabled for a language).